tests/functional/object/mcg/test_mcg_performance_profiles.py

```
@@ -0,0 +1,478 @@
+        noobaa_ocp = OCP(
+            kind="NooBaa",
+            namespace=config.ENV_DATA["cluster_namespace"],
+            resource_name="noobaa",
+        )
+        noobaa_profile = noobaa_ocp.get()["spec"].get("performanceProfile")
+        assert noobaa_profile == profile, (
+            f"Profile '{profile}' not propagated to NooBaa CR, "
+            f"got '{noobaa_profile}' instead"
+        )
+
+        logger.info(f"Profile '{profile}' successfully set and propagated to NooBaa CR")
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)